api/base/views.py

```diff
@@ -75,3 +76,6 @@ def root(request, format=None):
             'users': absolute_reverse('users:user-list'),
         }
     })
+
+def error_404(request, format=None, *args, **kwargs):
+    return HttpResponse('{"errors":[{"detail":"Not found."}]}', status=404, content_type='application/json')
```

Comment:
The content type needs to include `application/vnd.api+json`. You should be able to pass

```python
'application/vnd.api+json; application/json'
```